### ISSUE TITLE
[service] ERROR: Unable to find 'dependency' in remotes

### ISSUE THREAD

**Issue by ilya-lavrenov** [conan-io/conan-center-index#17940]
### What is your problem/feature request?

Hi,

Recently, the PR with adding a new hwloc recipe has been merged. 
I'm trying to submit another PR on top of it, but either locally or on [Conan CI](https://github.com/conan-io/conan-center-index/pull/17939) I experience the issue:
```
...
ERROR: Failed requirement 'hwloc/2.9.1' from 'onetbb/2021.8.0'
ERROR: Unable to find 'hwloc/2.9.1' in remotes
```
I suppose that some Conan registries are not updated yet and conan tool does not actually know about hwloc.

- how long does it take to update registries? Why is it not immediate and does not work as a post commit action? 
- can Conan CI understand that registry is not updated yet and build dependencies explicitly? Like locally I manually built hwloc using `conan create recipes/hwloc/all --version 2.9.1 -b missing` ? I saw you have API which extracts information from `conanfile.py` and it can be used to build dependencies. 

If we compare with `brew`:
- Once the PR is merged, artifacts of that PR are packed as bottles and during merge are published to file servers. So, users can immediately download new formula / updates.

Thanks.

**Comment by ilya-lavrenov** [comment]
any help?

**Comment by jcar87** [comment]
> how long does it take to update registries? Why is it not immediate and does not work as a post commit action?

The promotion process happens after a PR is merged in master, and in some cases it can take up to one hour until it is properly indexed and available. In a small number of cases we have seen in the past that if there is a CI issue during promotion, the recipe is updated but the packages are not uploaded.

I'm verifying what happened here and will relaunch your PR.
